rules/powerbi-template-app/rule.md

```diff
@@ -0,0 +1,100 @@
+---
+type: rule
+title: Do you create PowerBI Template Apps?
+uri: powerbi-template-apps
+authors:
+  - title: Isabel Sandstroem
+    url: https://www.ssw.com.au/people/isabel-sandstroem
+created: 2024-02-22T13:42:55.753Z
+guid: a3b2c4d5-e6f7-8910-gh11-i2j3k4l5m6n7
+---
+
+Creating PowerBI Template Apps is an efficient way to manage and distribute PowerBI reports and dashboards with other users.
+Once the Template App is created it will live in the [Power BI Apps](https://app.powerbi.com/groups/me/getapps/apps) marketplace. 
+Here a user can download the app to use on its own datasource.
```

Comment:
Maybe something like "Users have the ability to use the template app with their own data or use the default data provided"?